README.md

```diff
@@ -41,10 +41,14 @@ We need funds to do this.
 
 Transparency of Accounting is essential, this was an issue before.
 
+Stripe.com were contacted last year by Liam and said we could get a free "Non profit" account for 091 labs if we share our constituion. This would be a good route for collecting membership fees and then publishing csv exports online.
+
 ### Constitution
 
 We need to review and update constitution as appropriate.
-`
+
+Last years's version and the previous one are all here in [Liam's Google Drive](https://drive.google.com/open?id=0Bygd3T-0o7uAMkNjYmU2ZEpJb1k).
```

Comment:
Thanks for the link.
I'd like to start with a lighter weight constitution initially, this time.
Conscientious did not work out in practice; we never followed the procedures, e.g. publishing agendas in advance.
We can move to online conscientious building possibly, but I wouldn't want to bake it in to the constitution.
We had a serious issue with theft; cameras might well be the only deterrent.